File: HomeWork05/Commands/ControlllerCommands/MakeMainMenuCommand.java
package Commands.ControlllerCommands;
import Commands.ControlllerCommands.Base.ControllerCommand;
import Controllers.Controller;
import Models.Menu.Menu;
import Models.Menu.PunktMenu;


public class MakeMainMenuCommand extends ControllerCommand {
    public MakeMainMenuCommand(Controller controller) {
        super(controller);
    }

    @Override
    public void execute() {
        Menu menu = new Menu();
        menu.addPunkt(1,new PunktMenu("Загрузить справочник из файла", new LoadFromFilePhonebookCommand(controller)));
        menu.addPunkt(2,new PunktMenu("Сохранить справочник в файл", new SaveToFilePhonebookCommand(controller)));
        menu.addPunkt(0,new PunktMenu("Выход", new ExitAppCommand(controller)));
        controller.getScreen().setMenu(menu);
    }
    


    

    
}

File: HomeWork07/Commands/ControlllerCommands/MakeMainMenuCommand.cs
namespace HomeWork07;

public class MakeMainMenuCommand : ControllerCommand
{
    public MakeMainMenuCommand(Controller controller) :
        base(controller)
    {

    }

    
    public override void execute()
    {
        Menu menu = new Menu();
        menu.addPunkt(new PunktMenu(1, "Загрузить справочник из файла", new LoadFromFilePhonebookCommand(controller)));
        menu.addPunkt(new PunktMenu(2, "Сохранить справочник в файл", new SaveToFilePhonebookCommand(controller)));
        menu.addPunkt(new PunktMenu(0, "Выход", new ExitAppCommand(controller)));
        controller.getScreen().setMenu(menu);
    }

}